src/main/java/uk/ac/ebi/embl/converter/utils/ConversionEntry.java

```diff
@@ -10,29 +10,95 @@
  */
 package uk.ac.ebi.embl.converter.utils;
 
+import java.io.IOException;
+import java.io.StringReader;
+
 public class ConversionEntry {
     String sOID;
     String sOTerm;
     String definition;
     String feature;
-    String qualifier1;
-    String qualifier2;
+    String[] qualifier1;
+    String[] qualifier2;
```

Comment:
Suggestion to use a map instead of a String[]
`Map<String,String> qualifier1 = new HashMap<>();`

Then add all the qualifier key, values to the hash map. When there is no value add "true"